diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9c6f75575ff2c029ff2195615f83985bb6c9973e..ce0430858fd60268d37cd4a39390f0f83f0d4a75 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -3,7 +3,7 @@ import { Tapable } from 'tapable';
 import { Compiler } from 'webpack';
 export interface Options {
     readonly globPattern: string;
-    readonly postCssPlugins?: Array<PostCssPlugin<any>> | ((defaults: ReadonlyArray<PostCssPlugin<any>>) => PostCssPlugin<any>[]);
+    readonly postCssPlugins?: Array<PostCssPlugin> | ((defaults: ReadonlyArray<PostCssPlugin>) => PostCssPlugin[]);
     readonly camelCase?: boolean;
     readonly rootDir?: string;
     readonly searchDir?: string;
