diff --git a/node_modules/typed-css-modules-webpack-plugin/dist/index.d.ts b/node_modules/typed-css-modules-webpack-plugin/dist/index.d.ts
index 9c6f755..3c98288 100644
--- a/node_modules/typed-css-modules-webpack-plugin/dist/index.d.ts
+++ b/node_modules/typed-css-modules-webpack-plugin/dist/index.d.ts
@@ -1,15 +1,14 @@
 import { Plugin as PostCssPlugin } from 'postcss';
-import { Tapable } from 'tapable';
-import { Compiler } from 'webpack';
+import { Compiler, WebpackPluginInstance } from 'webpack';
 export interface Options {
     readonly globPattern: string;
-    readonly postCssPlugins?: Array<PostCssPlugin<any>> | ((defaults: ReadonlyArray<PostCssPlugin<any>>) => PostCssPlugin<any>[]);
+    readonly postCssPlugins?: Array<PostCssPlugin> | ((defaults: ReadonlyArray<PostCssPlugin>) => PostCssPlugin[]);
     readonly camelCase?: boolean;
     readonly rootDir?: string;
     readonly searchDir?: string;
     readonly outDir?: string;
 }
-export declare class TypedCssModulesPlugin implements Tapable.Plugin {
+export declare class TypedCssModulesPlugin implements WebpackPluginInstance {
     private dtsCreator;
     private useIncremental;
     private globPattern;
