diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9c6f75575ff2c029ff2195615f83985bb6c9973e..3c9828891c387ddf4b4c31a39e0b90ffecab2429 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,15 +1,14 @@
 import { Plugin as PostCssPlugin } from 'postcss';
-import { Tapable } from 'tapable';
-import { Compiler } from 'webpack';
+import { Compiler, WebpackPluginInstance } from 'webpack';
 export interface Options {
     readonly globPattern: string;
-    readonly postCssPlugins?: Array<PostCssPlugin<any>> | ((defaults: ReadonlyArray<PostCssPlugin<any>>) => PostCssPlugin<any>[]);
+    readonly postCssPlugins?: Array<PostCssPlugin> | ((defaults: ReadonlyArray<PostCssPlugin>) => PostCssPlugin[]);
     readonly camelCase?: boolean;
     readonly rootDir?: string;
     readonly searchDir?: string;
     readonly outDir?: string;
 }
-export declare class TypedCssModulesPlugin implements Tapable.Plugin {
+export declare class TypedCssModulesPlugin implements WebpackPluginInstance {
     private dtsCreator;
     private useIncremental;
     private globPattern;
