diff --git a/node_modules/webpack/types.d.ts b/node_modules/webpack/types.d.ts
index 0562921..57d0e0e 100644
--- a/node_modules/webpack/types.d.ts
+++ b/node_modules/webpack/types.d.ts
@@ -6635,8 +6635,8 @@ declare interface PerformanceOptions {
 	 */
 	maxEntrypointSize?: number;
 }
-declare interface Plugin {
-	apply: () => void;
+declare abstract class Plugin {
+	apply(compiler: Compiler): void;
 }
 declare interface PnpApiImpl {
 	resolveToUnqualified: (arg0: string, arg1: string, arg2?: any) => string;
@@ -10439,8 +10439,15 @@ declare namespace exports {
 		RuleSetUseItem,
 		Configuration,
 		WebpackOptionsNormalized,
-		WebpackPluginInstance
+		WebpackPluginInstance,
+		// MOD START: Backward compatibility
+		Plugin,
 	};
+
+	export namespace compilation {
+		export { Compilation };
+	}
+	// MOD END
 }
 
 export = exports;
